docker-compose.yml

```diff
@@ -1,10 +1,32 @@
 version: '3'
 
 services:
-  nodejs:
-    image: rapidpro-cht:1.0.0
-    container_name: rapidpro-cht
+  rapidpro2pg:
+    build: .
+    container_name: rapidpro2pg
     environment:
-      - pgUrl=postgres://couch2pg:secret=@localhost:5432/db'
-      - rapidProAuth=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
-    network_mode: "host"
\ No newline at end of file
+      POSTGRESQL_URL: ${POSTGRESQL_URL}
+      RAPIDPRO_URL: ${RAPIDPRO_URL}
+      RAPIDPRO_AUTH: ${RAPIDPRO_AUTH}
+    network_mode: "host"
+    command: node /usr/app/bin/index.js
+
+  postgres:
+    image: postgres:9.6
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_PASSWORD: $INT_PG_PASS
+      POSTGRES_USER: $INT_PG_USER
+      POSTGRES_DB: $INT_PG_DB
+      POSTGRES_HOST_AUTH_METHOD: "trust"
+
+  test:
+    build: .
+    volumes:
+      - ./src:/usr/app/src
+      - ./bin:/usr/app/bin
+      - ./tests:/usr/app/tests
+    depends_on:
+      - postgres
+    command: npm test
```

Comment:
I couldn't make the tests with docker to work. Unit test run OK but integration tests fail with:

```
  rapidpro2pg
    basic tests
      initial import into postgres
Error with migrations error: database "rapidpro2pgtest" does not exist
```

I created the image with `docker-compose build --build-arg node_version=12 test` and then run the tests with `docker-compose run test`.

I tried with and without setting up environment variables because according with the README default values are supplied. When used with env variables, these were the values:

```bash
export POSTGRESQL_URL="postgres://postgres:postgres@localhost:5432/postgres"
export RAPIDPRO_URL="http://localhost:5000/api"
export RAPIDPRO_AUTH="123456"
export INT_PG_USER="postgres"
export INT_PG_PASS="postgres"
export INT_PG_DB="postgres"
```

Anyway, when running the tests with docker-compose, shouldn't be any need to setup any variable, like when running the tests from medic-couch2pg, because postgres and couch instances are supplied by docker-compose, and env variables should be used to override the usage of those supplied services only.

Reply:
BTW, I think this is mostly because the way env variables are handled, in the case of medic-couch2pg, default values are provided by the `env` module, taking default values from [here](https://github.com/medic/cht-couch2pg/blob/master/.env), that are only red if not real env variables are set.

Reply:
I added an .env file to contain the default values for the integration tests. Can you please try again?